packages/shared/src/hooks/useLogin.ts

```diff
@@ -209,7 +197,7 @@ const useLogin = ({
     }),
     // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [session, queryEnabled, queryParams, hint, login, isLoading],
+    [queryEnabled, queryParams, hint, login, isLoading],
```

Comment:
Not blocking but since you did refactor things here might be good idea to also remove `useMemo` and add `useCallback` for `onSubmitSocialLogin` and `onSubmitPasswordLogin`

Reply:
Oh, yep, I agree. I have cleared this part. Seems my commit did not push through. Let me push it properly 😅